partcad-cli/src/partcad_cli/click/commands/init.py

```
@@ -7,30 +7,148 @@
 # Licensed under Apache License, Version 2.0.
 #
 import rich_click as click
-import os
+import os, sys
```

Ruff (0.8.2)

Multiple imports on one line

Split imports

(E401)